sema4ai/src/sema4ai_code/_lint.py

```diff
@@ -151,6 +151,28 @@ def __init__(
         self._weak_robocorp_language_server = weak_robocorp_language_server
         BaseLintInfo.__init__(self, lsp_messages, doc_uri, is_saved, weak_lint_manager)
 
+    @staticmethod
+    def _find_action_or_agent(uri: str) -> bool:
```

Comment:
This code seems to be a copy of `RobocorpLanguageServer._find_agent_package_spec_path`, please make that version a staticmethod and use it directly instead of copying it.